j5/backends/backend.py

```diff
@@ -152,7 +127,8 @@ class Environment:
     """
     A collection of board implementations that can work together.
 
-    Auto-populated with board mappings using metaclass magic.
+    Backends must be manually registered, but can this is a useful mechanism
```

Comment:
Not sure what you're trying to say here, could you try and make this clearer?

Reply:
Does e1291b3 help?